docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -235,28 +235,28 @@ https://kafka.apache.org/documentation/#security_authz_primitives[Authorization
 
 .Prerequisites
-* You've created a Kafka instance and the instance is in *Ready* state.
+* You've created a Kafka instance and the instance is in the *Ready* state.
 * The user account or service account that you're setting permissions for has been created in the organization.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. On the {service-url-kafka}[_Kafka Instances_^] page of the {product-kafka} web console, click the name of the Kafka instance that you want to set permissions for.
 . Click the *Access* tab to view the current ACL permissions for this instance.
 . Use this *Access* page to set permissions for a new account, add permissions to an existing account, or delete account permissions in this instance.
 +
 * To set permissions for a new account in this instance, follow these steps:
 
 .. Click *Manage access*.
-.. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
+.. In the *Account* list, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
 +
-If you don't see users in the drop-down list, ask your organization administrator to grant access to view other user accounts. For more information, see {base-url}{access-mgmt-url-kafka}#proc-user-account-access_managing-access[Allowing users to view other user accounts].
+If you don't see users in the *Account* list, ask your organization administrator to grant access to view other user accounts. For more information, see {base-url}{access-mgmt-url-kafka}#proc-user-account-access_managing-access[Allowing users to view other user accounts].
 .. Click *Next*.
 +
 --
 The *Review existing permissions* section lists any permission settings in this instance that are already defined for all accounts in the organization and for the same account that you previously selected, if applicable. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Access* page.
 
 If you previously selected a specific account, you can delete only permission entries that apply to individual accounts. If you previously selected *All accounts*, you can delete only permission entries that apply to all accounts.
 --
-.. Under *Assign Permissions*, use the drop-down menu to select and define the permissions for the specified account or all accounts for a resource type, such as a topic.
+.. From the list under *Assign Permissions*, select and define the permissions for the specified account or all accounts for a resource type, such as a topic.
```

Comment:
I found something slightly awkward about this rewrite when I read it (a couple of times). I think it might read a bit more naturally as "Under *Assign Permissions*, use the list to...."

Feel free to disagree, but for some reason I tripped over the rewritten version.

Reply:
Yes, I was trying not to use the word 'use', but it does read more naturally as it was.